### ISSUE TITLE
A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Screenreader : Incorrect state as 'Selected' is announced for controls such as "Create New Copespace", "Select Plan" and "Refresh" for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/vscode#106114]
"[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier."

### Environment Details:​
Application Name: VS Codespace
Application Version: 1.48.0-insider
Commit: 0a7364f00514c46c9caceece15e1f82f82e3712f
Date: 2020-07-22T05:43:52.797Z
Electron: 9.1.0
Chrome: 83.0.4103.122
Node.js: 12.14.1
V8: 8.3.110.13-electron.0
OS: Darwin x64 19.6.0
Operating System: MAC 10.15.6
Screen Reader: Voiceover
Standard Violated: WCAG 1.3.1, WCAG 4.2.1

### Repro Steps:​
1. Open VS Code insider 
2. Navigate till "Extension" button and search for the "Visual Studio Online"
3. Click on "Visual Studio Online" and navigate till "Remote Explorer" ,click on it.
4. Navigate the controls such as "Create New Copespace", "Select Plan" and "Refresh" using the keyboard.
5. Observe if the 'selected' state is announced for the screen reader users or not.

### Actual:
When screen reader users navigate to the controls such as "Create New Copespace", "Select Plan" and "Refresh" the 'selected' state is announced which is incorrect as visually the control is not selected so incorrect information is announced for screen reader users.

### Expected: ​
The 'selected' state for the controls such as "Create New Copespace", "Select Plan" and "Refresh" should not be announced to the screen reader users.

### ​User Impact:
If incorrect information regarding the state of the control is announced to the screen reader users then they will get confused and this leads to missing understand the content.

### Attachment:
[A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Screenreader.zip](https://github.com/microsoft/vscode/files/5174433/A11y_VSCodespaceBenchmark_Defaultsettingfornewcodespaces_Showpricinginfo_Screenreader.zip)

**Comment by isidorn** [comment]
Accessibility issue for extensions should be filled in the repository of the extension. Once that is done the extension author should investigate and file an issue in VS Code using proper description of our components. Thanks!

**Comment by MayuriNaik** [comment]
@isidorn We have already logged this bug for VS Codespace and they have redirected us to log an external bug for VS Code. So we logged this bug here.
